theoriq/dialog/code_items.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Annotated, Optional
+
+from pydantic import Field, field_validator, model_validator
+
+from .block import BaseData, BlockBase
+
+
+class CodeData(BaseData):
+    code: str
+    language: Optional[str] = None
+
+    def to_str(self) -> str:
+        if self.language:
+            result = [f"```{self.language}", self.code, "```"]
+        else:
+            result = ["```", self.code, "```"]
```

Comment:
This should do:

```suggestion
        result = [f"```{self.language or ''}", self.code, "```"]
```